website/src/views/Profile/Profile.test.jsx

```diff
@@ -1,29 +1,239 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
 import firebase from 'firebase';
+import {
+  shallow,
+  configure,
+} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import {
+  Button,
+} from 'antd';
 import { Profile } from './Profile';
 import '../../utils/tests/test.css';
 
-describe('Profile View', () => {
-  it('Expect the Profile page to match snap shot', () => {
-    jest.spyOn(firebase, 'auth')
-      .mockImplementation(() => ({
-        currentUser: {
-          displayName: 'testDisplayName',
-          email: 'test@test.com',
-          emailVerified: true,
-          getIdToken: () => new Promise(((resolve) => {
-            setTimeout(() => {
-              resolve(1234);
-            }, 0);
-          })),
-        },
-      }));
+configure({ adapter: new Adapter() }); // configure
 
+describe('Profile View when userData is undefined', () => {
+  it('Expect the Profile page to match snap shot', () => {
     const component = renderer.create(
       <Profile />,
     );
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
 });
+
+describe('Profile View when userData is defined as student. Empty university with empty requested Class', () => {
+  it('Expect the Profile page to match snap shot', () => {
+    const wrapper = shallow(<Profile />);
+    const userData = {
+      requested: {
```

Comment:
For these empty objects close the object on the same line. so it should become `requested: { }` same goes for universities

Reply:
got it!